model/parVarSheet.cc

```diff
@@ -359,7 +450,17 @@ namespace
                 else y0+=4.1*rowHeight;   
                 iC++;
                
-              }
+              } else if (auto p=it->plotWidgetCast())
+		       {
+				cairo::CairoSave cs(cairo);   
+				if (it==itemFocus) {
+					cairo_translate(cairo,xItem,yItem);  		    				   
+                    itemCoords.erase(itemFocus);   
+                    itemCoords.emplace(make_pair(itemFocus,make_pair(xItem,yItem)));         
```

Comment:
I'm guessing that itemCoords is a map (or similar) type. Simplest code would be

itemCoords[itemFocus]=make_pair(xItem,yItem);